modules/dag-l0/src/main/scala/org/tessellation/dag/l0/domain/snapshot/programs/Download.scala

```diff
@@ -121,23 +120,30 @@ object Download {
         val (lastSnapshot, lastContext) = result
 
         fetchSnapshot(none, lastSnapshot.ordinal.next).flatMap { snapshot =>
-          lastSnapshot.toHashed[F].flatMap { hashed =>
-            Applicative[F].unlessA {
-              Validator.isNextSnapshot(hashed, snapshot.value)
-            }(InvalidChain.raiseError[F, Unit])
-          } >>
+          hasherSelector
+            .forOrdinal(lastSnapshot.ordinal) { implicit hasher =>
```

Comment:
Makes sense, and seems to match other places where we get hasher for `lastSnapshot`